src/QrScan.css

```diff
@@ -1,6 +1,5 @@
 .qr {
   video {
     border: 2px solid rgb(200, 200, 200);
-    margin: 1.5em auto;
```

Comment:
So that's pretty interesting - it seems that it should only add margin from the top and bottom, while the left/right margin shoudl be left to auto.
Having this black bar might suggest that the parent container has just higher width than the video itself.

Have you tried with `margin: 1.5em 0;`? Can you check what is the width of parent container?

Reply:
`margin: 1.5em 0;` works fine! what boggles my mind is that the styling cannot be found in the output. I think react-qr-reader might be reading the margin value, removing it and incorporating it in the canvas/video? not sure.

it's also possible that we have some global css in Fether that makes this behave differently than normal. what's even more funny is that this margin doesn't seem to affect everybody (Luke on Fether doesn't have the margin/bar, testing the same source code)

testing with `size` prop set to `100`:

![image](https://user-images.githubusercontent.com/5739676/51032322-db048280-159f-11e9-970d-2539fef26882.png)

all parents have computed width `100px`. note that the video output is offset to the right; the light is at the center of the webcam, so it's really as if there was a margin to the left, and because of this the video output is cropped to its container (right part is lost) (however, manually offsetting the `left` css property of video doesn't bring the right part back)

Reply:
Interesting it might be related to hardware acceleration of the video and different GPUs, if removing the marign works we should merge as is.

Reply:
Ok :) Should I bump the version in the PR? Does the npm package update need to be done manually?

Reply:
Seems that there is no script to bump or auto-publish. So please bump the version and we can publish manually.